.github/workflows/testing.yml

```diff
@@ -72,6 +75,12 @@ jobs:
           # If we did not exit earlier, raise an error here
           exit 1
           ls
+      - name: Upload Results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{github.job}}-${{ github.run_number }}
```

Comment:
Is there a reason that once there is a space after the `{{` and once there is not?

Reply:
seems not to be the case, i'll fix it.